Tests/MessageGenerationTests.swift

```diff
@@ -225,4 +225,54 @@ class MessageGenerationTests: XCTestCase {
             XCTAssertEqual(message, expected[index], "Index: \(index).")
         }
     }
+
+    func testMessageFormatting() {
+        // For a given day, we'll always get the same message format.
+        // For a given day and location, always the same message.
```

Comment:
a) This is dependent on whether the user is in the northern or southern hemisphere, so it might be worth noting which you're testing here. 
b) Since you're only testing the message format here, do you need to mention the message itself?

Reply:
`his is dependent on whether the user is in the northern or southern hemisphere, so it might be worth noting which you're testing here.`

That's where the location part comes in, right? I should add some southern hemisphere tests as well.

Reply:
Right but you're testing the format for the northern hemisphere, not the message for any particular place in the northern hemisphere

Reply:
I'm testing them against a precise location, otherwise the number values could differ. 🤔